client/src/components/Profile/EditProfile.tsx

```diff
@@ -0,0 +1,184 @@
+                  {fillMonth.map((el) => (
+                    <MenuItem key={el} value={el}>
+                      {el}
+                    </MenuItem>
+                  ))}
+                </Select>
+                <Select className={classes.dropDownDate} id="selectDate" value="0">
+                  <MenuItem value="0">Date</MenuItem>
+                  {fillDate.map((el) => (
+                    <MenuItem key={el} value={el}>
+                      {el}
+                    </MenuItem>
+                  ))}
+                </Select>
+                <Select className={classes.dropDownYear} labelId="label" id="select" value="10">
+                  <MenuItem value="10">Year</MenuItem>
+                  {fillYear.map((el) => (
+                    <MenuItem key={el} value={el}>
+                      {el}
+                    </MenuItem>
+                  ))}
+                </Select>
+              </Box>
+              <TextField
+                className={classes.testWidth}
+                id="username"
+                margin="normal"
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                InputProps={{
+                  classes: { input: classes.inputs },
+                }}
+                name="email"
+                autoComplete="email"
+                autoFocus
+                placeholder="Your email"
+              />
+              <Box flexDirection="row" className={classes.phoneDetails}>
+                <span className={classes.phoneNumber}>No Phone number entered</span>
+                <Button className={classes.phoneButton}>Add a phone number</Button>
+              </Box>
+              <TextField
+                className={classes.testWidth}
+                id="username"
+                margin="normal"
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                InputProps={{
+                  classes: { input: classes.inputs },
+                }}
+                name="city"
+                autoComplete="city"
+                autoFocus
+                placeholder="Your Address"
+              />
+              <TextField
+                className={classes.testWidth}
+                id="username"
+                margin="normal"
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                InputProps={{
+                  classes: { input: classes.inputsDescription },
+                }}
+                name="description"
+                autoComplete="description"
+                autoFocus
+                placeholder="Your Description"
+              />
+            </Box>
+          </Grid>
+        </Grid>
+        <Box textAlign="center" marginTop={4}>
+          <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
+            Save
+          </Button>
+        </Box>
+      </form>
```

Comment:
You can align all the input labels to left for a better look

Reply:
Done the changes

Reply:
The link to a ticket is present in the PR Description.